app/webpacker/styles/application.scss

```diff
@@ -72,3 +72,12 @@ $govuk-image-url-function: frontend-image-url;
 .app-no-wrap {
   white-space: nowrap;
 }
+
+.app-survey-container {
+  background-color: govuk-colour("light-grey");
+  padding: 15px;
+  margin-top: -15px;
```

Comment:
```suggestion
  padding: 20px;
  margin-top: -20px;
```

![Screen Shot 2021-03-03 at 11 40 07](https://user-images.githubusercontent.com/319055/109800593-493faf80-7c15-11eb-938e-d08ecd7410b0.png)

Reply:
(That's with the smaller heading too)